docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -18,6 +18,7 @@ items:
 - apiVersion: nmstate.io/v1
   kind: NodeNetworkConfigurationPolicy
   spec:
+    desiredState:
       interfaces:
```

Comment:
@archana203 This doc update is from a bug you submitted: https://issues.redhat.com/browse/OSPRH-8467

Reply:
Lgtm